content/participants/ttjc@saifeemustafaq/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Mustafa Saifee
+type: "participant"
+name: "Mustafa Saifee"
+portfolio: "https://www.saifeemustafa.com/"
+github-link: "https://github.com/saifeemustafaq/"
+linkedin-link: "https://www.linkedin.com/in/saifeemustafaq/"
+looking-for: "internship"
+description: "Microsoft Azure | Python | DevOps"
+
+------
+
+> Hi! I am a Mustafa Saifee, let me walk you through my intro, projects and blogs.
```

Comment:
minor typo remove 'a'

Reply:
Hi I am a Mustafa

- Are you any non living thing or a human?? What do you mean "a" Mustafa

Reply:
Instead of I am **a** Mustafa --->  it should be I am Mustafa

Reply:
Haha. I am so sorry. I completely missed it. Thankyou for pointing that and sorry for that silly mistake

Reply:
@gkanishk I have done the suggested changes. Thank you